requirements/edx/base.txt

```diff
@@ -66,23 +66,23 @@ django-method-override==1.0.4  # via -r requirements/edx/base.in
 django-model-utils==4.0.0  # via -r requirements/edx/base.in, django-user-tasks, edx-bulk-grades, edx-celeryutils, edx-completion, edx-enterprise, edx-milestones, edx-organizations, edx-proctoring, edx-rbac, edx-submissions, edx-when, edxval, ora2, super-csv
 django-mptt==0.11.0       # via -r requirements/edx/base.in, django-wiki
 django-multi-email-field==0.6.1  # via edx-enterprise
-django-mysql==3.6.0       # via -r requirements/edx/base.in
+django-mysql==3.7.0       # via -r requirements/edx/base.in
 django-oauth-toolkit==1.3.2  # via -r requirements/edx/base.in
 django-object-actions==2.0.0  # via edx-enterprise
 django-pipeline==1.7.0    # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in
 django-pyfs==2.1          # via -r requirements/edx/base.in
 django-ratelimit-backend==2.0  # via -r requirements/edx/base.in
 django-require==1.0.11    # via -r requirements/edx/base.in
 django-sekizai==1.1.0     # via -r requirements/edx/base.in, django-wiki
-django-ses==0.8.14        # via -r requirements/edx/base.in
+django-ses==1.0.0         # via -r requirements/edx/base.in
```

Comment:
This version has some breaking changes: https://github.com/django-ses/django-ses/releases. 
`#183 Upgrade to boto3 (replacing boto). Remove setting for AWS_SES_PROXY and related settings. You must use environment variables to configure the proxy now. This is documented in the README.
`

Is that a problem?

Reply:
I don't see any problem with it, a few other packages in base.in are already using boto3. but @jmbowman can tell us for sure.

Reply:
I looked at that yesterday, but didn't see anything that should affect us.  We don't use a proxy for AWS, and none of the other listed changes seemed to impact us either.  Seems low enough risk to just try deploying it, we can revert if it causes unexpected issues.

Reply:
Good. I'll merge it then.